charts/capi-runtime-extensions/values.yaml

```diff
@@ -2,30 +2,39 @@
 # SPDX-License-Identifier: Apache-2.0
 
 hooks:
-  CalicoCNI:
-    defaultPodSubnet: 192.168.0.0/16
-    crsStrategy:
-      defaultTigeraOperatorConfigMap:
-        name: tigera-operator
-      defaultInstallationConfigMaps:
-        DockerCluster:
+  cni:
+    calico:
+      defaultPodSubnet: 192.168.0.0/16
+      crsStrategy:
+        defaultTigeraOperatorConfigMap:
+          name: tigera-operator
+        defaultInstallationConfigMaps:
+          DockerCluster:
+            create: true
+            configMap:
+              name: calico-cni-crs-installation-dockercluster
+              content: ""
```

Comment:
Could not find references to `content` used to create helm installation configmaps.

Reply:
Used in https://github.com/d2iq-labs/capi-runtime-extensions/blob/39ceadb8b958bcca0efeed7c983c675d330184cd/charts/capi-runtime-extensions/templates/cni/calico/manifests/docker/crs-installation.yaml#L12